runtime/compiler/runtime/RelocationRuntimeLogger.cpp

```diff
@@ -240,10 +209,16 @@ TR_RelocationRuntimeLogger::relocationTime()
 
       bool wasLocked = lockLog();
       method(false);
-      JITRT_PRINTF(jitConfig())(jitConfig(), " <%p-%p> ",
-                                             reloRuntime()->exceptionTable()->startPC,
-                                             reloRuntime()->exceptionTable()->endPC);
-      JITRT_PRINTF(jitConfig())(jitConfig(), " Time: %d usec\n", reloEndTime-_reloStartTime);
+
+      rtlogPrintf(jitConfig(), reloRuntime()->fej9()->_compInfoPT,
+                  " <%p-%p> ",
+                  reloRuntime()->exceptionTable()->startPC,
+                  reloRuntime()->exceptionTable()->endPC);
```

Comment:
`startPC` and `endPC` are `UDATA` here as well

Reply:
Addressed in https://github.com/eclipse-openj9/openj9/pull/14210/commits/de354ea9c291db3231788240d621ed75f430322c